src/AssemblySharedInfoGenerator/AssemblyInfoGenerator.csproj

```diff
@@ -32,4 +32,8 @@
   </ItemGroup>
   <Import Project="Sdk.targets" Sdk="Microsoft.NET.Sdk" />
   <Import Project="$(MSBuildExtensionsPath)\Microsoft\VisualStudio\v$(VisualStudioVersion)\TextTemplating\Microsoft.TextTemplating.targets" />
+  <Target Name="T4TemplateGen" BeforeTargets="Build">
+    <CallTarget Targets="TransformAll" />
+    <Message Importance="high" Text="Running Template Generator for AssemblyInfoProperties" />
```

Comment:
For some reason using the `TransformOnBuild` property does not work well with skd style projects and incremental builds.
The target `T4TemplateGen` will call the T4Generator target `TransformAll` every time a Build is triggered but not if the build is skipped due to the project being up to date